indico/modules/events/payment/controllers.py

```diff
@@ -109,6 +110,15 @@ def _process(self):
         form = self.plugin.event_settings_form(prefix='payment-', obj=defaults, plugin_settings=plugin_settings)
         if can_modify and form.validate_on_submit():
             self.plugin.event_settings.set_multi(self.event, form.data)
+            match plugin_event_settings.get('enabled'), form.enabled.data:
+                case True, False:
+                    self.event.log(EventLogRealm.management, LogKind.negative, 'Payment',
```

Comment:
Maybe at some pint the module names could become an enum as well?

Reply:
Not sure, we have MANY different ones... I guess we could use an enum for the string, but I would not store it as an enum in the DB, since that'd make adding new ones a bit of a pain.

Reply:
I suggested an enum just to avoid accidentally creating similar-but-different categories like `Payment` and `Payments`, I wouldn't want to enforce it at the DB-level but just to make it easier for us to know what categories already exist :)